spec/round_spec.rb

```diff
@@ -0,0 +1,88 @@
+require './lib/deck'
+require './lib/card'
+require './lib/round'
+require './lib/turn'
+
+RSpec.describe Round do
+  let(:card_1) { Card.new("What is the capital of Alaska?", "Juneau", :Geography) }
+  let(:card_2) { Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM) }
+  let(:card_3) { Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM) }
+
+  let(:cards) { [card_1, card_2, card_3] }
+
+  let(:deck) { Deck.new(cards) }
+  
+  let(:round) { Round.new(deck) }
+
+  it "round to hold deck" do
+    expect(round.deck).to eq(deck) 
+  end
+
+  it "sets no turns as default" do
+    expect(round.turns).to be_empty
+  end
+  
+  it "returns the current card" do
+    expect(round.current_card).to eq(card_1)
+  end
+  
+  it "takes a turn with the current card" do
+    expect(round.take_turn("Juneau").guess).to eq(Turn.new("Juneau", card_1).guess)
+  end
+
+  it "can evaluate correct guesses" do
+    expect(round.take_turn("Juneau").correct?).to be(true)
```

Comment:
An alternative to this (and some would argue cleaner) way to write this could be:

expect(round.take_turn("Juneau").to be_correct

Reply:
does this check the "correct?" method automatically?

Reply:
yes! this is thanks to Rspec's built-in predicate matcher.